README.md

```diff
@@ -116,6 +117,10 @@ because it creates a shadow copy.
 
 Deeply compares passed params.
 
+### [findLast](./lib/find-last.ts)
+
+Returns the desired value from the array, searching from the right.
```

Comment:
Let's use the same description MDN uses, but with slight changes: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

> Returns the value of the last element in the provided array that satisfies the provided testing function. Otherwise `undefined`.